bin/dm_parse_Nback.py

```diff
@@ -0,0 +1,119 @@
+"""
+Parse the N-back eprime text files into BIDS tsvs
+
+Usage:
+  parse_eprime_Nback.py [options] <text_file>
+
+Arguments:
+    <text_file>           The location of the ePRIME text file to parse.
+
+Options:
+  -o, --output FILE        Name of output csv file
+                           [default: Nback_results.tsv]
+  --debug                  Debug logging in Erin's very verbose style
+  -n,--dry-run             Dry run
+  --help                   Print help
+
+DETAILS
+Parse the N-back eprime text files for RTMS in the of scanner.
+Written by Erin W Dickie, November 18 2015
+Modified to extract event timings by Jerry Jeyachandra, November 2018
+Modified to parse task file to bids tsv format by Thomas Tan, Aug 2020
+"""
+from docopt import docopt
+import numpy as np
+import os
+import pandas as pd
+import subprocess
+import glob
+import sys
+
+arguments       = docopt(__doc__)
+inputfile        = arguments['<inputfile>']
+resultsfile      = arguments['--output']
+DEBUG           = arguments['--debug']
+DRYRUN          = arguments['--dry-run']
+
+if DEBUG : print(arguments)
+
+corr = {'c':1,'0':0, '1':1, '3':1}
+
+## loop over the inputfiles
+
+if DEBUG: print("\n file: {}".format(inputfile))
+
+with open(inputfile, "r", encoding='utf-16') as myfile:
+    filetext=myfile.read()
+
+## find and replace this weird text string that show up between every character
+filetext = filetext.replace('\x00','')
+filetext = filetext.replace('\r\n','\n') ##a windows formatting thing..
+
+## split all the text into individual trials using "LogFrame End" text string
+## this string is printed to the log at the conclusion of everytrial
+trials = filetext.split('LogFrame End')
+
+#Create a list to store dictionaries
+message_df_dict = []
+
+## loop over individual trials
+trial_count = 0
+for trial in trials:
+
+    # for each trial collect the CorrectResponse, Subject's Response and the Reation Time
+    CorrectResponse = int()
+    ReactionTime = int()
+    SubjectResponse = int()
+    TrailType = str()
+    TrailTime = int()
+
+    for line in trial.split('\n\t'):
+        if ("Running:" in line) :
+            TrailType = line.split(':')[1].replace(" ",'')
+        if ("letterdisp.OnsetTime:" in line) :
+            TrailTime = line.split(':')[1].replace(" ",'')
+        if ("resp:" in line) and ("letterdisp.RESP" not in line) and ("letterdisp.CRESP" not in line):
+            CorrectResponse = line.split(':')[1].replace(" ",'')
+        if (("letterdisp.RT" in line) and ("letterdisp.RTTime" not in line)):
+            ReactionTime = line.split(':')[1].replace(" ",'')
+        if "letterdisp.RESP" in line:
+            SubjectResponse = line.split(':')[1].replace(" ",'')
+
+    #Skip if list type
+    if (TrailType == 'List1') or (TrailType == ''):continue
```

Comment:
```suggestion
    if (TrialType == 'List1') or (TrialType == ''):continue
```

Indent would be needed here as well